rwgame/RWImGui.cpp

```diff
@@ -0,0 +1,204 @@
+#include "RWImGui.hpp"
+
+#include <ai/CharacterController.hpp>
+#include <objects/CharacterObject.hpp>
+#include <objects/VehicleObject.hpp>
+
+#include "RWGame.hpp"
+
+#include <imgui.h>
+#include <imgui_impl_sdl.h>
+#include <imgui_impl_opengl3.h>
+
+#include <glm/glm.hpp>
+
+#include <gl/gl_core_3_3.h>
+#include <glm/gtx/norm.hpp>
+
+namespace {
+void WindowDebugStats(RWGame& game) {
+    auto& io = ImGui::GetIO();
+
+    auto time_ms = 1000.0f / io.Framerate;
```

Comment:
Imho 22-40 should be moved more into engine. (some opcodes uses it)

Reply:
Uses what?

Reply:
Stats about frames. For example opcode 0078 needs delta time of avg frame.